### PR TITLE
Adding Project Subdirectories

### PR DESCRIPTION
**Pull request by dltm-markwan**
## What are you changing in this pull request and why?
Adding Project Subdirectories. I have had multiple clients ask what the Project Directory option is and how to use it. 

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/docs/build/projects.md [hunk 1]
```diff
@@ -52,6 +52,26 @@ Edit `dbt_project.yml` to set up common project configurations such as:
 
 For complete details on project configurations, see [dbt_project.yml](/reference/dbt_project.yml).
 
+## Project subdirectories
+
+The Project Subdirectory option in dbt Cloud allows you to specify a subdirectory within your git repository that dbt should use as the root directory for your project. This can be useful in situations where you have multiple dbt projects within a single repository, or when you want to organise your dbt project files into subdirectories for easier management.
```
Comment (mirnawong1) [inline website/docs/docs/build/projects.md]:
```suggestion
You can use the Project subdirectory option in dbt Cloud to specify a subdirectory in your git repository that dbt should use as the root directory for your project. This is helpful when you have multiple dbt projects in one repository or when you want to organize your dbt project files into subdirectories for easier management.
```

File: website/docs/docs/build/projects.md [hunk 2]
```diff
@@ -52,6 +52,26 @@ Edit `dbt_project.yml` to set up common project configurations such as:
...
+
+To use the Project Subdirectory option in dbt Cloud, follow these steps:
```
Comment (mirnawong1) [inline website/docs/docs/build/projects.md]:
```suggestion
To use the Project subdirectory option in dbt Cloud, follow these steps:
```

File: website/docs/docs/build/projects.md [hunk 3]
```diff
@@ -52,6 +52,26 @@ Edit `dbt_project.yml` to set up common project configurations such as:
...
+
+1. **Click** the Settings dropdown on the top navigation menu and **Click** Account Settings
```
Comment (mirnawong1) [inline website/docs/docs/build/projects.md]:
```suggestion
1. Click on the cog icon on the upper right side of the page and click on **Account Settings**.
```

File: website/docs/docs/build/projects.md [hunk 4]
```diff
@@ -52,6 +52,26 @@ Edit `dbt_project.yml` to set up common project configurations such as:
...
+<img width="379" alt="project-subdirectory-01" src="https://user-images.githubusercontent.com/114556261/228104923-591cd791-6f38-4a10-a983-7d85cd18f0b2.png">
```
Comment (mirnawong1) [inline website/docs/docs/build/projects.md]: i can't see the image here. to add images:
- use `<Lightbox src="/img/docs/your_image-name.jpg" title="Add your title here"/> `
- in that same branch, add your images probably in this [location](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/static/img/docs/building-a-dbt-project) of the docs repo (please use jpg)

File: website/docs/docs/build/projects.md [hunk 5]
```diff
@@ -52,6 +52,26 @@ Edit `dbt_project.yml` to set up common project configurations such as:
...
+
+2. **Click** on a Project to configure Project Subdirectory
```
Comment (mirnawong1) [inline website/docs/docs/build/projects.md]:
```suggestion
2. Under **Projects**, select the project you want to configure as a project subdirectory.
```

File: website/docs/docs/build/projects.md [hunk 6]
```diff
@@ -52,6 +52,26 @@ Edit `dbt_project.yml` to set up common project configurations such as:
...
+
+3. **Click** on Edit on the sliding pane
```
Comment (mirnawong1) [inline website/docs/docs/build/projects.md]:
```suggestion
3. Select **Edit** on the lower right-hand corner of the page.
```

File: website/docs/docs/build/projects.md [hunk 7]
```diff
@@ -52,6 +52,26 @@ Edit `dbt_project.yml` to set up common project configurations such as:
...
+<img width="1409" alt="project-subdirectory-02" src="https://user-images.githubusercontent.com/114556261/228104897-954e280b-cb68-4f7c-be3a-ceba5d9b90e3.png">
+
+4. **Enter** the name of the subdirectory in Project Subdirectory. For example, if your dbt project files are located in a subdirectory called `<repository>/finance`, you would enter `finance` as the subdirectory.
```
Comment (mirnawong1) [inline website/docs/docs/build/projects.md]:
```suggestion
4. In the **Project subdirectory** field, add the name of the subdirectory. For example, if your dbt project files are located in a subdirectory called `<repository>/finance`, you would enter `finance` as the subdirectory. 
```

Comment (mirnawong1) [reply website/docs/docs/build/projects.md]: love this example ✨

File: website/docs/docs/build/projects.md [hunk 8]
```diff
@@ -52,6 +52,26 @@ Edit `dbt_project.yml` to set up common project configurations such as:
...
+1. **Click** the Settings dropdown on the top navigation menu and **Click** Account Settings
+<img width="379" alt="project-subdirectory-01" src="https://user-images.githubusercontent.com/114556261/228104923-591cd791-6f38-4a10-a983-7d85cd18f0b2.png">
+
+2. **Click** on a Project to configure Project Subdirectory
+
+3. **Click** on Edit on the sliding pane
+<img width="1409" alt="project-subdirectory-02" src="https://user-images.githubusercontent.com/114556261/228104897-954e280b-cb68-4f7c-be3a-ceba5d9b90e3.png">
+
+4. **Enter** the name of the subdirectory in Project Subdirectory. For example, if your dbt project files are located in a subdirectory called `<repository>/finance`, you would enter `finance` as the subdirectory. 
+You can also reference nested subdirectories. For example, if your dbt project files are located in `<repository>/teams/finance`, you would enter `teams/finance` as the subdirectory. Please note: you do not need leading or trailing `/` in the Project Subdirectory parameter. **Click** the Save button when complete.
```
Comment (mirnawong1) [inline website/docs/docs/build/projects.md]:
```suggestion
* You can also reference nested subdirectories. For example, if your dbt project files are located in `<repository>/teams/finance`, you would enter `teams/finance` as the subdirectory. Please note: you do not need leading or trailing `/` in the Project subdirectory parameter. 

5. Click **Save** when you've finished.
```

File: website/docs/docs/build/projects.md [hunk 9]
```diff
@@ -52,6 +52,26 @@ Edit `dbt_project.yml` to set up common project configurations such as:
...
+<img width="660" alt="project-subdirectory-03" src="https://user-images.githubusercontent.com/114556261/228117931-f2651529-ebf3-4f19-bd33-14e299810a6f.png">
+
+Once you have configured the Project Subdirectory option, dbt Cloud will use the specified subdirectory as the root directory for your dbt project. This means that any dbt commands that you run in dbt Cloud, such as dbt run or dbt test, will operate on the files within the subdirectory that you specified. If there is no `dbt_project.yml` file detected in the Project Subdirectory, you will be prompted to initialise the dbt project.
```
Comment (mirnawong1) [inline website/docs/docs/build/projects.md]:
```suggestion
After configuring the Project subdirectory option, dbt Cloud will use it as the root directory for your dbt project. This means that dbt commands, such as `dbt run` or `dbt test`, will operate on files within the specified subdirectory. If there is no dbt_project.yml file in the Project subdirectory, you will be prompted to initialize the dbt project.
```

**Comment by mirnawong1** [comment]
hey @dbt-markwan this is so awesome 😍 - really love the detail and clear language! thank you for opening this pr up, and I've left some suggestions like changing the heading to sentence case vs title case (Hello world vs Hello World).

In terms of the images, so great that you've added some. I'm not sure if we need images for all the steps but if you'd like to add them:

- in this same branch, add your images probably in this [location](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/static/img/docs/building-a-dbt-project) of the docs repo (please use jpg)
- add or use <Lightbox src="/img/docs/your_image-name.jpg" title="Add your title here"/> in the .md file

File: website/docs/docs/build/projects.md (hunk 4 above)
Comment (mirnawong1) [inline website/docs/docs/build/projects.md]: sugg removing img for this step

```suggestion
```

File: website/docs/docs/build/projects.md [hunk 10]
```diff
@@ -52,6 +52,26 @@ Edit `dbt_project.yml` to set up common project configurations such as:
...
+<img width="379" alt="project-subdirectory-01" src="https://user-images.githubusercontent.com/114556261/228104923-591cd791-6f38-4a10-a983-7d85cd18f0b2.png">
+
+2. **Click** on a Project to configure Project Subdirectory
+
+3. **Click** on Edit on the sliding pane
+<img width="1409" alt="project-subdirectory-02" src="https://user-images.githubusercontent.com/114556261/228104897-954e280b-cb68-4f7c-be3a-ceba5d9b90e3.png">
+
+4. **Enter** the name of the subdirectory in Project Subdirectory. For example, if your dbt project files are located in a subdirectory called `<repository>/finance`, you would enter `finance` as the subdirectory. 
+You can also reference nested subdirectories. For example, if your dbt project files are located in `<repository>/teams/finance`, you would enter `teams/finance` as the subdirectory. Please note: you do not need leading or trailing `/` in the Project Subdirectory parameter. **Click** the Save button when complete.
+<img width="660" alt="project-subdirectory-03" src="https://user-images.githubusercontent.com/114556261/228117931-f2651529-ebf3-4f19-bd33-14e299810a6f.png">
```
Comment (mirnawong1) [inline website/docs/docs/build/projects.md]:
```suggestion
<Lightbox src="/img/docs/building-a-dbt-project/project-subdirectory.jpg" width="75%"  title="Entering `finance` in the Project subdirectory field."/>
```

File: website/docs/docs/build/projects.md [hunk 11]
```diff
@@ -52,6 +52,26 @@ Edit `dbt_project.yml` to set up common project configurations such as:
...
+
+To use the Project Subdirectory option in dbt Cloud, follow these steps:
+
+1. **Click** the Settings dropdown on the top navigation menu and **Click** Account Settings
+<img width="379" alt="project-subdirectory-01" src="https://user-images.githubusercontent.com/114556261/228104923-591cd791-6f38-4a10-a983-7d85cd18f0b2.png">
+
+2. **Click** on a Project to configure Project Subdirectory
+
+3. **Click** on Edit on the sliding pane
+<img width="1409" alt="project-subdirectory-02" src="https://user-images.githubusercontent.com/114556261/228104897-954e280b-cb68-4f7c-be3a-ceba5d9b90e3.png">
```
Comment (mirnawong1) [inline website/docs/docs/build/projects.md]: sugg removing this screenshot

```suggestion
```

**Comment by mirnawong1** [comment]
hey @dbt-markwan this is so awesome 😍 - really love the detail and clear language! thank you for opening this pr up, and I've left some suggestions like changing the heading to sentence case vs title case (Hello world vs Hello World).

In terms of the images, so great that you've added some but i think it's failing because they need to be added to our static/img/docs folder. I fixed the last screenshot for you as i think it'll be helpful to show a correct and completed example for users. I'm not sure if we need images for all the steps but if you'd like to add them:

- in this same branch, add your images probably in this [location](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/static/img/docs/building-a-dbt-project) of the docs repo (please use jpg)
- add or use <Lightbox src="/img/docs/your_image-name.jpg" title="Add your title here"/> in the .md file

**Review by mirnawong1 — APPROVED** [review]
this looks so good, what a wonderful addition @dbt-markwan - thanks for making the docs better! ready for you to merge ✨

**Comment by dltm-markwan** [comment]
How exciting!

**Pull request merged**